src/__tests__/ParseObject-test.js

```diff
@@ -845,6 +845,49 @@ describe('ParseObject', () => {
     expect(o.op('count')).toBe(undefined);
   });
 
+  it('can revert a specific field in unsaved ops', () => {
+    var o = ParseObject.fromJSON({
+      className: 'Item',
+      objectId: 'canrevertspecific',
+      count: 5
+    });
+    o.set({ cool: true });
+    o.increment('count');
+    expect(o.get('cool')).toBe(true);
+    expect(o.get('count')).toBe(6);
+    o.revert('cool');
+    expect(o.get('cool')).toBe(undefined);
+    expect(o.op('cool')).toBe(undefined);
+    expect(o.get('count')).toBe(6);
+    expect(o.op('count')).not.toBe(undefined);
+  });
+
+  it('can revert multiple fields in unsaved ops', () => {
+    var o = ParseObject.fromJSON({
+      className: 'Item',
+      objectId: 'canrevertmultiple',
+      count: 5,
+      age: 18,
+      gender: 'female'
+    });
+    o.set({ cool: true, gender: 'male' });
+    o.increment('count');
+    o.increment('age');
+    expect(o.get('cool')).toBe(true);
+    expect(o.get('count')).toBe(6);
+    expect(o.get('age')).toBe(19);
+    expect(o.get('gender')).toBe('male');
+    o.revert('age', 'count', ['gender']);
```

Comment:
This is really non sensical to allow such things. It’s not because it’s ‘supported’ that it should be allowed.